get.sh

```diff
@@ -413,10 +449,12 @@ getBinaryOpenjdk()
 					jar_dir_name=${jar_dir_array[0]}
 					if [[ "$jar_dir_name" =~ "test-image" ]] || [[ "$jar_dir_name" =~ "tests-" ]]; then
 						if [ "$jar_dir_name" != "openjdk-test-image" ]; then
-							mv $jar_dir_name ../openjdk-test-image
+							moveDirectorySafely $jar_dir_name ../openjdk-test-image
```

Comment:
I'm curious as to how we're ending up the situation where this function is required, as it seems to add to the compexity somewhat. If we really want to catch the condition where it already exists can we just use somethign like:
```suggestion
							[ ! -r ../openjdk-test-image ] && mv "$jar_dir_name" ../openjdk-test-image || exit 1
```
(If you really need a message that should be easy to add)

Reply:
catching the condition is key, as I don't think when it was originally added as to the side effect of a new type of archive being added, and trashing a different archives content

Reply:
no I mean "brevity" as in: concise and exact use of words in writing or speech.
ie.easy to understand
one liners I find complex and hard to understand as they typically use options or commands i've never heard of or use, like "-r"
also a function keeps the logic the same in the several places it is used, rather than having to change in each line.

I understand this is subjective... but it's the way I prefer

Reply:
I would argue that `if [[ `ls -d */ | wc -l` -eq 1 ]] && [[ `find . -maxdepth 1 -mindepth 1 | wc -l` -eq 1 ]]; then` is particularly unclear and hard to follow exactly what it's doing, whereas `-r` to test if something is present is far more common, but as you say it's subjective ;-)